src/pages/index.tsx

```diff
@@ -68,6 +74,47 @@ const Home = ({ seoMeta }: PageProps) => {
         borderColour={2}
       />
       <CanYouBeAProgrammer />
+      <FlexContainer className='bg-[#923CFF] w-full p-4 flex-col  gap-4'>
+        <FlexContainer className='w-full lg:w-[45%] max-w-[350px] md:max-w-[500px]'>
+          <Image
+            src={COMMUNITY_SECTION.communityBannerImage}
+            fullHeight={true}
+            fullWidth={true}
+            alt='community'
+          />
+        </FlexContainer>
+        <FlexContainer className='max-w-[350px] text-white gap-1  md:max-w-[500px]'>
+          <Text level='h1' className='font-bold text-2xl'>
+            {COMMUNITY_SECTION.communityTitleLine}
+          </Text>
+          <Text level='p' className='text-center'>
+            {COMMUNITY_SECTION.communityDescriptionLine}
+          </Text>
+        </FlexContainer>
+        <FlexContainer className='max-w-[350px] flex-col text-white gap-1  md:max-w-[500px]'>
+          <Text level='h1' className='font-bold text-2xl'>
+            In Community, You can
+          </Text>
+          <FlexContainer className='flex-col gap-3 '>
+            {COMMUNITY_SECTION.goalOfCommunity.map((goal, i) => (
+              <Text
+                level='h1'
+                key={i}
+                className='w-full bg-white text-black rounded-[5px] text-center px-5 py-2 font-bold'
+              >
+                {goal}
+              </Text>
+            ))}
```

Comment:
This can be separated as a Pill. Check if Pill already exists or not. Type of the pill will be Ghost Pill as it doesn't have any color in BG.